.gitlab-ci.yml

```diff
@@ -11,14 +11,15 @@ include:
 #
 - /.gitlab-ci/stages/prepare.yml
 - /.gitlab-ci/stages/quality.yml
+- /.gitlab-ci/stages/register.yml
 
 image: node:10-alpine
 
 stages:
 - "Prepare"
 - "Code Quality"
 - "Send Code Quality Reports"
-- "Build and Push Docker Image"
+- "Registration"
```

Comment:
`Docker Image Registration` ?

Reply:
can be changed as you want in the future.
I merge it for now =P